src/internal/clusterstate/v2.12.0/pjs.go

```diff
@@ -42,14 +42,13 @@ func createPJSSchema(ctx context.Context, env migrations.Env) error {
 		);
 		-- PJS has a server side cache for jobs. Utilizing this cache allows PJS to skip subtrees when spawning job trees.
 		CREATE TABLE pjs.job_cache (
-			job_hash BYTEA NOT NULL,
-			job_id BIGINT REFERENCES pjs.jobs(id) ON DELETE CASCADE,
+			job_id BIGINT REFERENCES pjs.jobs(id) ON DELETE CASCADE PRIMARY KEY,
```

Comment:
A stub necessitates that we may write an entry to this table without the job hash being there.